docs/LabVIEW_Debugging_Techniques.md

```diff
@@ -0,0 +1,128 @@
+|System Channel   |Description                                                               |
++=================+==========================================================================+
+|HP Count         |The number of times the Primary Control Loop reported being late.         |
++-----------------+--------------------------------------------------------------------------+
+|HP Loop Duration |The duration of the Primary Control Loop in nanoseconds.                  |
++-----------------+--------------------------------------------------------------------------+
+|LP Count         |The number of times the Data Processing Loop reported being late.         |
++-----------------+--------------------------------------------------------------------------+
+|Model Count      |The number of times the models have not completed their execution in time.|
++-----------------+--------------------------------------------------------------------------+
+```
+ 
+If the value of the count channels increase over time, the execution host is not achieving the desired loop rates. You can use the system channels in conjunction with an **[alarm](https://zone.ni.com/reference/en-XX/help/372846M-01/veristand/add_alarms/)** or **[procedure](https://www.ni.com/documentation/en/veristand/latest/manual/add-configure-procedure/)** to handle the event.
+
+#### System Monitor Add-on
+
+The VeriStand System Monitor is a Custom Device that tracks memory resources and CPU usage on an RT target running the VeriStand Engine. Set the update rate (Hz) in System Explorer to determine how often the custom device checks CPU and memory usage and sends them to the corresponding channel. The VeriStand System Monitor can only be used on an RT target. The custom device returns an error if you target it to a Windows system.
+
+#### Real-Time Trace Viewer
+
+VeriStand 2020 provides built-in support for using the **[Real-Time Trace Viewer](https://zone.ni.com/reference/en-XX/help/370715P-01/lvtracehelp/lv_tracetoolkit_help/)** and **[Real-Time Trace Viewer VIs](https://zone.ni.com/reference/en-XX/help/370715P-01/lvtrace/tracetoolkitvis_pal/)** to capture the timing and execution data of VI and thread events for applications running on an RT target. In LabVIEW, select **Tools»Real-Time Module»Trace Viewer** to display the Real-Time Trace Viewer. For more info on how to use Real-Time Trace Viewer, in LabVIEW go to **Help>>LabVIEW Help>>Real-Time Module>>Real-Time Trace Viewer**.
+
+
+#### Additional Debugging Options for VeriStand
+Upon request, National Instrument may provide advanced debugging tools to help you resolve certain custom device issues. These tools are a last resort when all other debugging options have been exhausted. Please contact National Instruments for more information.
+ 
+
+#### Table of Debugging and Benchmarking Techniques
+
+
+```eval_rst
++--------------------+-------------------+-----------+-----------------------------------------------------------------+
```

Comment:
The Contents of this table should be updated with the info from here: https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-benchmark-debug/